raft.go

```diff
@@ -1452,7 +1452,7 @@ func (r *Raft) requestVote(rpc RPC, req *RequestVoteRequest) {
 	if lastVoteTerm == req.Term && lastVoteCandBytes != nil {
 		r.logger.Info("duplicate requestVote for same term", "term", req.Term)
 		if bytes.Compare(lastVoteCandBytes, req.Candidate) == 0 {
-			r.logger.Warn("duplicate requestVote from", "candidate", req.Candidate)
+			r.logger.Warn("duplicate requestVote from", "candidate", r.trans.DecodePeer(req.Candidate))
```

Comment:
It looks like [something similar](https://github.com/hashicorp/raft/blob/e51cd9f557cc5e879cd8e41f4054725a41f262d0/raft.go#L1417) is done a couple lines up - could we use the `candidate` value instead?

Reply:
Good catch, sure thing!